app/templates/response/add_file.html

```diff
@@ -2,23 +2,25 @@
       enctype="multipart/form-data">
     <div class="add-file col-sm-9">
         <div class="fileupload-control">
+            <fieldset>
```

Comment:
This was added because `<legend>` tags should only be present within `<fieldset>` tags... **This still has to be done for the other Add Response Tabs**